packages/plugin/vite/test/ViteConfig_spec.ts

```diff
@@ -0,0 +1,8 @@
+// import { expect } from 'chai';
+
+// import { VitePluginConfig } from '../src/Config';
+// import ViteConfigGenerator from '../src/ViteConfig';
+
+describe('ViteConfigGenerator', () => {
+  // TODO
```

Comment:
Are we looking to do this before this PR lands or is this a longterm TODO?

Reply:
Yep, this should be considered a long-term plan. But I will still write plat of the test for this PR.